src/pipx/venv.py

```diff
@@ -274,6 +274,34 @@ def install_package(
                 wrap_message=False,
             )
 
+    def install_unmanaged_packages(
+        self, requirements: List[str], pip_args: List[str]
+    ) -> None:
+        """Install packages in the venv, but do not record them in the metadata."""
+
+        # TODO: Do we need this?
+        # check syntax and clean up spec and pip_args
+        # (package_or_url, pip_args) = parse_specifier_for_install(
```

Comment:
Probably users will need to pass pip arguments?

Reply:
They can do so. The comment is referring to the `parse_specifier_for_install` call, which was copied from `install_package`.

None of the things that call does seem relevant to me in this context (a set of requirements listed in a script):
* Strip any markers (e.g. python_version > "3.4")
* Ensure `--editable` is removed for any package_spec not a local path
* Convert local paths to absolute paths

I think what I should do is just remove that comment as resolved now.

Reply:
No problem, you were right to pick up on the no longer relevant TODO.